packages/engine/src/actions/getFromObject.js

```diff
@@ -0,0 +1,44 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { applyArrayIndices, get, serializer, type } from '@lowdefy/helpers';
```

Comment:
@Gervwyk I don't think we need a `getFromObject` method, since the object isn't a Lowdefy data object

Reply:
Not sure I understand. So no `getFromObject` just use `get` instead?

Reply:
Not sure I understand. What about `getGlobal` for example?

Reply:
Sorry I was confused here